File: js/task3a/S3_OBJ_04.js
function omit(obj, keys) {
  return Object.keys(obj).reduce((result, key) => {
    if (!keys.includes(key)) {
      result[key] = obj[key];
    }
    return result;
  }, {});
}

const user = {
  id: 1,
  name: "Ahmet",
  email: "ahmet@test.com",
  password: "123456",
  role: "admin"
};

const result = omit(user, ["password", "email"]);
console.log(result);
console.log(user);

File: python/task3a/S3_OBJ_04.py
def omit(obj, keys):
  return {k: v for k, v in obj.items() if k not in keys}


user = {
  "id": 1,
  "name": "Ahmet",
  "email": "ahmet@test.com",
  "password": "123456",
  "role": "admin"
}

result = omit(user, ["password", "email"])

print(result)
print(user)
